.vscode/launch.json

```diff
@@ -18,7 +18,8 @@
         "order": 1
       },
       "env": {
-        "DEBUG_VSCODE_JAVA":"true"
+        "DEBUG_VSCODE_JAVA":"true",
+        "DEBUG_START_BUILD_SERVER": "true"
```

Comment:
When `DEBUG_START_BUILD_SERVER == false`, it will use BuildShip to build the project. I add this for debug purpose.